bundle/bind.go

```diff
@@ -44,8 +44,12 @@ func (e *executor) Bind(
 
 	go func() {
 		e.actionStarted()
-		// Create namespace name that will be used to generate a name.
-		ns := fmt.Sprintf("%s-%.4s-", instance.Spec.FQName, bindAction)
+		ns := instance.Context.Namespace
+		// Determine if we should be using the context namespace from the executor config.
+		if !e.doNotCreateNS {
```

Comment:
This is confusing phrasing with the double negative. I would do:

```
if e.doNotCreateNS {
ns := instance.Context.Namespace
else {
ns := createdNamespace
}
```

Less mental gymnastics and more direct.

Reply:
I prefer either: `createNS` OR `skipCreateNS`. Using skip allows you to use the Golang default of False for bools and to me it's easier to read then `doNot`